partials/tabs/tryonline.html

```diff
@@ -1,43 +1,205 @@
             </div>
+          </div>
+          <div class="col m8">
+            <ul class="left no-margin">
+              <li ng-repeat="(k,v) in val">
+                <div class="input-field no-margin">
+                <input type="text" ng-model="sections[section]['bears'][bear][k]">
+                <label>{{k}}</label>
+                </div>
+              </li>
+            </ul>
+          </div>
+          <div class="col m1">
+            <i ng-click="remove_bears(section, bear)" class="fa fa-minus-circle black-text bears-icon" aria-hidden="true">
+            </i>
+          </div>
+        </div>
+      </div>
+
+      <div class="row blue-grey lighten-5 black-text new-section z-depth-1">
+        <div class="col m3 offset-m3">
+          <input type="text" placeholder="Give a section name!" class="no-margins" id="new-section">
         </div>
-        <div class="center">Warming up the bears!</div>
+        <div class="col m6">
+            <a ng-click="add_sections()" class="btn waves-effect">
+              <span class="footer-btn-text">NEW SECTION</span>
+              <i class="fa fa-plus white-text bears-icon" aria-hidden="true"></i>
+            </a>
+        </div>
+      </div>
     </div>
+  </div>
+
+  <div ng-show="sections" class="row">
+    <div class="col m12 s12 center">
+      <div>
+        <a ng-click="run_coala()" class="center run-coala waves-effect waves-light btn">RUN 
+          <img src="https://api.coala.io/en/latest/_static/images/coala_logo.svg"/>
+        </a>
+      </div>
+      <br>
+      <div ng-show="running_coala" class="preloader-wrapper small active">
+        <div class="spinner-layer spinner-blue">
+          <div class="circle-clipper left">
+            <div class="circle"></div>
+          </div><div class="gap-patch">
+            <div class="circle"></div>
+          </div><div class="circle-clipper right">
+            <div class="circle"></div>
+          </div>
+        </div>
+        <div class="spinner-layer spinner-red">
+          <div class="circle-clipper left">
+            <div class="circle"></div>
+          </div><div class="gap-patch">
+            <div class="circle"></div>
+          </div><div class="circle-clipper right">
+            <div class="circle"></div>
+          </div>
+        </div>
+        <div class="spinner-layer spinner-yellow">
+          <div class="circle-clipper left">
+            <div class="circle"></div>
+          </div><div class="gap-patch">
+            <div class="circle"></div>
+          </div><div class="circle-clipper right">
+            <div class="circle"></div>
+          </div>
+        </div>
+        <div class="spinner-layer spinner-green">
+          <div class="circle-clipper left">
+            <div class="circle"></div>
+          </div><div class="gap-patch">
+            <div class="circle"></div>
+          </div><div class="circle-clipper right">
```

Comment:
General Formatting: Use a new line for every block, list, or table element, and indent every such child element: Move the opening "div" to its own line.

*HTMLLintBear, severity MAJOR, section `html`.*